content/docs/reference/config/installation-options-changes/index.md

```diff
@@ -1,13 +1,13 @@
 ---
 title: Installation Options Changes
-description: Details the Helm chart installation options differences between release-1.0 and release-1.1.
+description: Details the Helm chart installation options differences between release-1.1 and release-1.2.
 weight: 30
 keywords: [kubernetes, helm, install, options]
 ---
 
-The tables below show changes made to the installation options used to customize Istio install using Helm between release 1.0 and release 1.1. The tables are grouped in to three different categories:
+The tables below show changes made to the installation options used to customize Istio install using Helm between release 1.1 and release 1.2. The tables are grouped in to three different categories:
 
-- The installation options already in the previous release but whose values or descriptions have been modified in the new release.
+- The installation options already in the previous release but whose values have been modified in the new release.
```

Comment:
@johnma14 Are you no longer including the options where the descriptions changed? Why?

Reply:
@rcaballeromx  If values remain the same and only the descriptions change, we decided not to track that as a change here(You can see discussion around this in: https://github.com/istio/istio.io/pull/4402. This was done to put focus on the changes to the values and avoid the noise around documenting these options). This file will be used only to track changes in the values of configuration options, new options added and old ones that are removed. Hope this helps.